compiler/src/iree/compiler/Codegen/ExternalInterfaces/Utils.h

```diff
@@ -37,17 +45,21 @@ struct PackedLayoutMaterializerAttrExternalModelBase
   getEncodingInfo(Attribute attr, RankedTensorType type) const {
     const EncodingPackedLayoutMaterializerAttr *impl =
         static_cast<const EncodingPackedLayoutMaterializerAttr *>(this);
+    IREE::Codegen::MaterializeEncodingInfo info;
     // If the layout is already resolved, use it directly.
     if (auto config = impl->getConfiguration(attr)) {
       if (auto namedAttr = config.getNamed(kEncodingInfoAttrName)) {
-        std::optional<IREE::Codegen::MaterializeEncodingInfo> info =
+        std::optional<IREE::Codegen::MaterializeEncodingInfo> maybeInfo =
```

Comment:
side question that is not related to this PR: are we able to remove the `std::optional` given that we have the verification today?

Reply:
Sorry, I was looking at a different function that returns std::optional<IREE::Codegen::MaterializeEncodingInfo>, however, here, for deserializeEncodingInfo it also can't be avoided because the verifier itself relies on this function.